applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -1117,6 +1117,25 @@ void ConstitutiveLawUtilities<3>::CalculateRotationOperatorVoigt(
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<SizeType TVoigtSize>
+void ConstitutiveLawUtilities<TVoigtSize>::CalculateVonMisesEquivalentStress(
+    const array_1d<double, VoigtSize>& rStressVector,
+    const Vector& rStrainVector,
+    double& rEquivalentStress
+    )
+{
+    double I1, J2;
+    array_1d<double, VoigtSize> deviator = ZeroVector(VoigtSize);
+
+    ConstitutiveLawUtilities<VoigtSize>::CalculateI1Invariant(rStressVector, I1);
```

Comment:
those two functions should also be templated in the vector_type

Reply:
changing